lib/cli/init/template/package.json

```diff
@@ -7,15 +7,14 @@
     "build": "npm run build:js && npm run build:sass && npm run build:translate",
     "build:js": "mkdir -p ./public/js && browserify ./assets/js/index.js > ./public/js/bundle.js",
     "build:sass": "mkdir -p ./public/css && npm-sass ./assets/scss/app.scss > ./public/css/app.css",
-    "build:translate": "hof-transpiler ./apps/**/translations/src -w --shared ./node_modules/hof-template-partials/translations",
+    "build:translate": "hof-transpiler ./apps/**/translations/src -w",
     "prepublish": "npm run build",
     "prestart": "npm run build:translate"
   },
   "dependencies": {
     "hof-behaviour-summary-page": "^2.0.0",
-    "hof-bootstrap": "^10.0.0",
-    "hof-frontend-toolkit": "^1.0.1",
-    "hof-template-partials": "^3.2.0",
```

Comment:
I'm guessing you know what you're doing, but I'm just checking because I don't know. You meant to remove `hof-template-partials`?